src/main/java/mekanism/common/content/gear/IBlastingItem.java

```diff
@@ -0,0 +1,13 @@
+package mekanism.common.content.gear;
+
+import java.util.Map;
+import net.minecraft.core.BlockPos;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.state.BlockState;
+
+public interface IBlastingItem {
+
+    Map<BlockPos, BlockState> getBlastedBlocks(Level world, Player player, ItemStack stack, BlockPos pos, BlockState state);
```

Comment:
This seems to be nullable so should be annotated as such, but is there a reason implementors don't just return `Collections#emptyMap` instead of null?

Reply:
I was using null to indicate that it hadn't blasted as their logic was sometimes different if that was the case, but empty set would work I guess.

Reply:
Either is probably fine, just if it can return nullable it should at least be annotated.